omero/sysadmins/whatsnew.txt

```diff
@@ -7,7 +7,27 @@ What's new for OMERO 5.3 for sysadmins
   the Ice Python bindings if your package manager does not provide the Ice
   Python packages.
 
-- All OMERO web apps can now be installed using ``pip install``.
+- :doc:`version-requirements` has been updated to reflect other changes in
+  version support for 5.3.0 and tentative plans for 5.4.0.
+  
+- All official OMERO.web apps can now be installed from PyPI. You should
+  reinstall your plugins when you upgrade.
 
-- Windows support discontinued, see :doc:`/sysadmins/windows-migration`.
+- Windows support has been discontinued, see
+  :doc:`/sysadmins/windows-migration`.
 
+- OMERO.web has been decoupled from the server and can now be deployed
+  separately. Support for deployment using Apache has been dropped; if your
```

Comment:
requirements is python 2.7 
That will imply some work for sysadmin still on centos6